package.json

```diff
@@ -51,10 +51,12 @@
     "@mui/x-data-grid": "^6.5.0",
     "@reduxjs/toolkit": "^1.9.5",
     "@tanstack/react-query": "^4.29.12",
+    "@types/lodash": "^4.14.195",
```

Comment:
please put this to `dev-dependencies`

Read [this](https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencie) to know what different